src/basic/log.c

```diff
@@ -588,6 +592,27 @@ static int log_do_header(
         return 0;
 }
 
+static void log_do_context(struct iovec *iovec, size_t iovec_len, size_t *n) {
+        assert(iovec);
+        assert(n);
+
+        if (iovec_len <= 1)
+                return;
```

Comment:
Is this check needed? We don't expect such short iovec, and the check below should handle it if we get one.